index.html

```diff
@@ -152,5 +152,57 @@ <h2 class="about_headline">About</h2>
         </li>
       </ul>
     </section>
+
+    <!-- Contact section -->
+
+    <section id="contact_section">
+      <h2 class="contact_title">Get started</h2>
+      <p class="contact_text">
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
+        tempor incididunt ut labore et dolore magna.
+      </p>
+      <form class="form" action="https://formspree.io/f/mzboznlq" method="POST">
+        <input
+          class="input1"
+          type="text"
+          name="name"
+          maxlength="30"
+          placeholder="Name"
+          required
+        />
+        <input
+          class="input2"
+          type="email"
+          name="email"
+          placeholder="exemple@mail.com"
+          required
+        />
+        <textarea
+          class="input3"
+          name="message"
```

Comment:
[OPTIONAL]: Please ensure you stick to the design as much as possible (e.g., font, colors, images, tests, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

## Your design:
![your-design](https://user-images.githubusercontent.com/10332499/148947585-18ffccf4-f039-44dc-bb78-cc1db683bb60.PNG)

## Figma design: 
![figma](https://user-images.githubusercontent.com/10332499/148947619-0d17d3cc-4ed8-471c-bde2-81b9e7257aee.PNG)